src/behaviors/focusZone.ts

```diff
@@ -311,6 +312,11 @@ function shouldIgnoreFocusHandling(keyboardEvent: KeyboardEvent, activeElement:
   return false
 }
 
+export const isActiveDescendantAttribute = 'data-is-active-descendant'
+export const activeDescendantActivatedDirectly = 'activated-directly'
```

Comment:
Would this be a good place to include a comment describing what `activated-directly` means?

Reply:
Yeah, that's a good idea.  Added some comments in https://github.com/primer/components/pull/1308/commits/0a6057c65a41d71d7f57f711803a7d2a6853c289